src/Routes/Tasks.js

```diff
@@ -0,0 +1,59 @@
+const BaseRoute = require('../Structure/BaseRoute');
+const handleError = require('../Util/handleError');
+
+class TasksRoute extends BaseRoute {
+    constructor(client, db, jobs) {
+        super('/tasks');
+        this.router = require('express').Router();
+        this.client = client;
+        this.db = db;
+        this.jobs = jobs;
+        this.routes();
+    }
+
+    routes() {
+        this.router.use(this.requiresAuth.bind(this), this.isMod.bind(this));
```

Comment:
Move this into the `get` route as the `post` route already has its own auth middleware defined.